src/EloquentDataTable.php

```diff
@@ -166,7 +166,9 @@ protected function resolveRelationColumn(string $column): string
         $columnName = array_pop($parts);
         $relation = str_replace('[]', '', implode('.', $parts));
 
-        if ($this->isNotEagerLoaded($relation)) {
+        if (! $relation) {
+            return $this->query->getModel()->getTable().'.'.$column;
```

Comment:
I think this will fail in cases where we use a table alias?

Reply:
Good catch, I will try to check this

Reply:
Should be fixed now.
I just noticed a `addTablePrefix()` method was already created for this purpose.
So I just reused it.